Description: Allow to pass in configure args.
 Allow to pass in configure args.
Forward: pending
Author: Jan Engelhardt <jengelh@medozas.de>
Last-Update: 2012-02-14
---
 nx-X11/lib/X11/Imakefile          |    6 ++++--
 nx-X11/programs/Xserver/Imakefile |    4 +++-
 2 files changed, 7 insertions(+), 2 deletions(-)

Index: nx-X11/lib/X11/Imakefile
===================================================================
--- a/nx-X11/lib/X11/Imakefile
+++ b/nx-X11/lib/X11/Imakefile
@@ -1147,15 +1147,17 @@ ks_tables.h: $(XINCLUDESRC)/keysymdef.h
 
 includes:: ks_tables.h
 
+CONFIGURE ?= ./configure
+
 #if NXLibraries
 
 $(NX_XCOMPCONFIGTARGET):
 	cd ../../../nxcomp && \
-	./configure
+	${CONFIGURE}
 
 $(NX_XCOMPEXTCONFIGTARGET):
 	cd ../../../nxcompext && \
-	./configure
+	${CONFIGURE}
 
 #ifdef SunArchitecture
 $(NX_XCOMPLIBTARGET): $(NX_XCOMPCONFIGTARGET)
Index: nx-X11/programs/Xserver/Imakefile
===================================================================
--- a/nx-X11/programs/Xserver/Imakefile
+++ b/nx-X11/programs/Xserver/Imakefile
@@ -1036,9 +1036,11 @@ NX_XSHADOWLIBDIR    = $(XTOP)/../nxcomps
 NX_XSHADOWLIBTARGET = $(NX_XSHADOWLIBDIR)/$(NX_XSHADOWLIBNAME)
 NX_XSHADOWCONFIGTARGET = $(NX_XSHADOWLIBDIR)/config.status
 
+CONFIGURE ?= ./configure
+
 $(NX_XSHADOWCONFIGTARGET):
 	cd $(NX_XSHADOWLIBDIR) && \
-	./configure
+	${CONFIGURE}
 
 #ifdef SunArchitecture
 $(NX_XSHADOWLIBTARGET): $(NX_XSHADOWCONFIGTARGET)
